### PR TITLE
Specific assay files

### PR DESCRIPTION
**Pull request by jen-dfci**
Changes to reflect the new CDS direct export button.  Also, removed duplicate documentation in sequencing data access page.

### CODE REVIEW

**Comment by jen-dfci** [comment]
@adamjtaylor Added you to review language regarding entityIDs.  If someone else should be tagged instead, let me know.

**Review by alexeliotlash — APPROVED** [review]

**Comment by jen-dfci** [comment]
@alexeliotlash I added one change since you approved this request -- realized I should also update "Content Updates" on the main readme.

@adamjtaylor I will wait to merge until you have had a chance to look at the entityID language.

**Review by alexeliotlash — APPROVED** [review]

File: data_submission/clin_biospec_assay.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ As stated in [Data Submission Overview](../data_submission/overview.md), data su
 1. Uploading assay data files to Synapse; and
 2. Completing and validating metadata using the Data Curator App (DCA).
 
+!!! Once assay data files are submitted to Synapse, the files will have entityIDs (e.g. syn12345670) assigned to them. These can then be prepopulated into the manifests on the DCA. For this reason, assay files should be submitted before generating the associated manifests.
```
Comment (adamjtaylor) [inline data_submission/clin_biospec_assay.md:11]: LGTM

**Review by adamjtaylor — APPROVED** [review]
Happy with that, incl the entityId line change

**Pull request merged**